backend/pkg/scheduler.go

```diff
@@ -92,25 +95,47 @@ func createReports() {
 	// save in a new lookup the shape:
 	// { {scheduleID}: excelFilePath }
 
-	// f := excelize.NewFile()
+	dashboard, err := api.GetDashboard("")
+
+	if err != nil {
+		log.DefaultLogger.Error(err.Error())
+	}
+
+	rawSQL := dashboard.GetRawSQL(2)
+	queryRequest, err := api.NewQueryRequest(rawSQL).ToRequestBody()
+
+	if err != nil {
+		log.DefaultLogger.Error(err.Error())
+		// return nil, err
+	}
+
+	response, err := http.Post("http://admin:admin@localhost:3000/api/tsdb/query", "application/json", queryRequest)
+
+	if err != nil {
+		log.DefaultLogger.Error(err.Error())
+		// return nil, err
+	}
+
+	qr, err := api.QueryFromResponse(response)
+
 	f, e := excelize.OpenFile("./data/test.xlsx")
 
 	if e != nil {
 		log.DefaultLogger.Error("erorr opening", e.Error())
-
 	}
 
-	// Create a new sheet.
-	index := f.NewSheet("Sheet2")
-	// Set value of a cell.
-	f.SetCellValue("Sheet1", "B2", 100)
-	f.SetCellValue("Sheet2", "A2", "Hello Tony.")
-
-	// Set active sheet of the workbook.
-	f.SetActiveSheet(index)
+	columns := []string{"A", "B", "C", "D", "E", "F", "G", "H", "I", "J", "K", "L", "M"}
+	ri := 0
+	for _, row := range qr.Results.A.Tables[0].Rows {
+		ci := 0
+		for range qr.Results.A.Tables[0].Columns {
+			cellRef := columns[ci] + strconv.Itoa(ri+1)
```

Comment:
Maybe overkill, but little algo to convert integer offset to excel column:

```
const getExcelColumn = offset => {
    var dividend = offset
    var columnName = ""
    var modulo = 0
    while (dividend > 0) {
        modulo = (dividend - 1) % 26
        columnName = String.fromCharCode(65 + modulo) + columnName
        dividend = Math.floor((dividend - modulo) / 26)
    }
    return columnName;
}
```

Not thoroughly tested, but seems to work!